tools/formatting/format-all

```diff
@@ -6,6 +6,8 @@
 # 0 on success
 # 2 is clang-format 8 could not be found 
 
+#TODO: Needs an update?
+
```

Comment:
@fsimonis Could you check this during the review?

Reply:
Removed it. The formatting scripts don't need an update.